sdk/core/Azure.Core.Experimental/tests/DynamicJsonTests.cs

```diff
@@ -225,6 +226,25 @@ public void CanSetArrayValueToNull()
             Assert.IsNull((int?)jsonData[0]);
         }
 
+        [Test]
+        public void CanEnumerateArrayWithProperCasing()
```

Comment:
Would you be willing to move this into the section of tests on casing?  (~line 478ish)